src/engine/enginebuffer.h

```diff
@@ -144,6 +145,7 @@ class EngineBuffer : public EngineObject {
     bool isTrackLoaded();
     TrackPointer getLoadedTrack() const;
 
+    double getExactPlayPos();
```

Comment:
wasn't sure of the naming here

Reply:
there is a `getVisualPlayPos`, maybe exact isn't the right term, but I think just naming it `getPlayPos` isn't as clear as it could be

Reply:
:man_shrugging: